python/lsst/meas/base/plugins.py

```diff
@@ -137,23 +137,39 @@ def __init__(self, config, name, schema, metadata):
         SingleFramePlugin.__init__(self, config, name, schema, metadata)
         self.keyProbability = schema.addField(name + "_value", type="D",
                                               doc="Set to 1 for extended sources, 0 for point sources.")
+        self.keyFlag = schema.addField(name + "_flag", type="Flag",
+                                              doc="Set to 1 for any fatal failure.")
```

Comment:
The indentation here is a bit weird -- it's not a fatal flaw, but it would be nicer if the "d" of "doc" appeared directly beneath the "n" of "name". In fact, the text is short enough that everything would fit on a single line, which would be better still.